Tests/Functional/MiscellaneousTests.swift

```diff
@@ -353,8 +353,7 @@ class MiscellaneousTestCase: XCTestCase {
     
     func testProductWithMissingModules() {
         fixture(name: "Miscellaneous/ProductWithMissingModules") { prefix in
-            XCTAssertBuilds(prefix)
-            XCTAssertFileExists(prefix, ".build/debug/libBar.a")
+            XCTAssertBuildFails(prefix)
```

Comment:
I've attempted to create a unit test, but it fails to build. See also https://github.com/Bouke/swift-package-manager/tree/SR-1188-unit-test.